Rules.md

```diff
@@ -0,0 +1,29 @@
+# 📌 Contribution Rules to Dev-Scripts
+
+- Every project must be included inside a particular **Domain Folder** ( like - Web Development,Python,Java,etc).
+
+   <img src="/assets/pic1.jpg" alt="Project domain" >
+
+- Inside every **Domain Folder** include your **Project folder with Project name**, inside that include your **Project Files** and **README.md** file.
+
+   <img src="/assets/pic2.jpg" alt="Project Files" >
+   
+   <img src="/assets/pic3.jpg" alt="Project Files" >
+
+- Every Project must include **README.md** file is **mandatory** and the following below points mentioned must be included in the **README.md** file:- 
+   1. Description
+   
+   2. Tech Stack
+   
+   3. Demonstration(include Demo video or GIF)
+  
+   4. Basic Component in Project.
+   
+   5. Show Steps on how to run in Local
+
+   6. If any further info contributors want to include, you are most welcome to do.
+
+  
+  
+  **Example:-** Domain Folder/ProjectName/Folder/Projectfiles and README.md files.
+  **If any of the above-mentioned rules are not fulfilled thn particular PR will not get merged until they are resolved**
```

Comment:
change thn spelling because after PR i need to make that changes and commit for that. 😅

Reply:
done